content/release-notes/1.61.0.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2022-01-28T10:21:28-07:00
+linktitle: "1.61.0"
+title: "1.61.0"
+kubernetes: "1.20, 1.21, and 1.22"
```

Comment:
We're supporting four versions right now since all four of these are in kURL and are [officially in support](https://kubernetes.io/releases/).
```suggestion
kubernetes: "1.20, 1.21, 1.22, and 1.23"
```